### DISCUSSION TITLE
Is it possible to forward slots to children?

### DISCUSSION THREAD

**Discussion by An-di-K** [lit/lit#4673] [Q&A] [answered]
Hello! 
I am very new to lit and am wondering if it possible to forwards named slots to child elements.
I haven't found anything about that in the lit documentation so I am assuming it is not possible. But I wanted to ask first before I give up that approach and instead try it via templates or properties.

I have created a simple example on the [lit playground](https://lit.dev/playground/#sample=examples%2Fhello-world&project=W3sibmFtZSI6InBhcmVudC1lbGVtZW50LnRzIiwiY29udGVudCI6ImltcG9ydCB7aHRtbCwgTGl0RWxlbWVudH0gZnJvbSAnbGl0JztcbmltcG9ydCB7Y3VzdG9tRWxlbWVudH0gZnJvbSAnbGl0L2RlY29yYXRvcnMuanMnO1xuaW1wb3J0IFwiLi9jaGlsZC1lbGVtZW50LmpzXCJcblxuQGN1c3RvbUVsZW1lbnQoJ3BhcmVudC1lbGVtZW50JylcbmV4cG9ydCBjbGFzcyBQYXJlbnRFbGVtZW50IGV4dGVuZHMgTGl0RWxlbWVudCB7XG4gIFxuICByZW5kZXIoKSB7XG4gICAgcmV0dXJuIGh0bWxgXG4gICAgICAgIDxwPlBhcmVudCBFbGVtZW50PC9wPlxuICAgICAgICA8Y2hpbGQtZWxlbWVudD48L2NoaWxkLWVsZW1lbnQ-XG4gICAgICAgIDwhLS0gPHNsb3QgbmFtZT1cInRlc3RcIj48L3Nsb3Q-IC0tPlxuICAgIGA7XG4gIH1cbn1cbiJ9LHsibmFtZSI6ImluZGV4Lmh0bWwiLCJjb250ZW50IjoiPCFET0NUWVBFIGh0bWw-XG48aGVhZD5cbiAgPHNjcmlwdCB0eXBlPVwibW9kdWxlXCIgc3JjPVwiLi9wYXJlbnQtZWxlbWVudC5qc1wiPjwvc2NyaXB0PlxuPC9oZWFkPlxuPGJvZHk-XG4gIDxwYXJlbnQtZWxlbWVudD5cbiAgICAgPGgxIHNsb3Q9XCJ0ZXN0XCI-VGVzdDwvaDE-XG4gIDwvcGFyZW50LWVsZW1lbnQ-XG48L2JvZHk-XG4ifSx7Im5hbWUiOiJwYWNrYWdlLmpzb24iLCJjb250ZW50Ijoie1xuICBcImRlcGVuZGVuY2llc1wiOiB7XG4gICAgXCJsaXRcIjogXCJeMy4wLjBcIixcbiAgICBcIkBsaXQvcmVhY3RpdmUtZWxlbWVudFwiOiBcIl4yLjAuMFwiLFxuICAgIFwibGl0LWVsZW1lbnRcIjogXCJeNC4wLjBcIixcbiAgICBcImxpdC1odG1sXCI6IFwiXjMuMC4wXCJcbiAgfVxufSIsImhpZGRlbiI6dHJ1ZX0seyJuYW1lIjoiY2hpbGQtZWxlbWVudC50cyIsImNvbnRlbnQiOiJpbXBvcnQge2h0bWwsIExpdEVsZW1lbnR9IGZyb20gJ2xpdCc7XG5pbXBvcnQge2N1c3RvbUVsZW1lbnR9IGZyb20gJ2xpdC9kZWNvcmF0b3JzLmpzJztcblxuQGN1c3RvbUVsZW1lbnQoJ2NoaWxkLWVsZW1lbnQnKVxuZXhwb3J0IGNsYXNzIENoaWxkRWxlbWVudCBleHRlbmRzIExpdEVsZW1lbnQge1xuICBcbiAgcmVuZGVyKCkge1xuICAgIHJldHVybiBodG1sYFxuICAgICAgICA8cD5DaGlsZCBFbGVtZW50PC9wPlxuICAgICAgICA8c2xvdCBuYW1lPVwidGVzdFwiPjwvc2xvdD5cbiAgICBgO1xuICB9XG59XG4ifV0)

**Comment by augustjk** [comment, marked as answer]
Put the `<slot>` element inside the child element tag in the parent template and also give it the slot attribute for the child.

Is this the behavior you're looking for? [playground](https://lit.dev/playground/#sample=examples%2Fhello-world&project=W3sibmFtZSI6InBhcmVudC1lbGVtZW50LnRzIiwiY29udGVudCI6ImltcG9ydCB7aHRtbCwgTGl0RWxlbWVudH0gZnJvbSAnbGl0JztcbmltcG9ydCB7Y3VzdG9tRWxlbWVudH0gZnJvbSAnbGl0L2RlY29yYXRvcnMuanMnO1xuaW1wb3J0IFwiLi9jaGlsZC1lbGVtZW50LmpzXCJcblxuQGN1c3RvbUVsZW1lbnQoJ3BhcmVudC1lbGVtZW50JylcbmV4cG9ydCBjbGFzcyBQYXJlbnRFbGVtZW50IGV4dGVuZHMgTGl0RWxlbWVudCB7XG4gIFxuICByZW5kZXIoKSB7XG4gICAgcmV0dXJuIGh0bWxgXG4gICAgICAgIDxwPlBhcmVudCBFbGVtZW50PC9wPlxuICAgICAgICA8Y2hpbGQtZWxlbWVudD5cbiAgICAgICAgICA8c2xvdCBuYW1lPVwidGVzdFwiIHNsb3Q9XCJ0ZXN0XCI-PC9zbG90PlxuICAgICAgICA8L2NoaWxkLWVsZW1lbnQ-XG4gICAgYDtcbiAgfVxufVxuIn0seyJuYW1lIjoiaW5kZXguaHRtbCIsImNvbnRlbnQiOiI8IURPQ1RZUEUgaHRtbD5cbjxoZWFkPlxuICA8c2NyaXB0IHR5cGU9XCJtb2R1bGVcIiBzcmM9XCIuL3BhcmVudC1lbGVtZW50LmpzXCI-PC9zY3JpcHQ-XG48L2hlYWQ-XG48Ym9keT5cbiAgPHBhcmVudC1lbGVtZW50PlxuICAgICA8aDEgc2xvdD1cInRlc3RcIj5UZXN0PC9oMT5cbiAgPC9wYXJlbnQtZWxlbWVudD5cbjwvYm9keT5cbiJ9LHsibmFtZSI6InBhY2thZ2UuanNvbiIsImNvbnRlbnQiOiJ7XG4gIFwiZGVwZW5kZW5jaWVzXCI6IHtcbiAgICBcImxpdFwiOiBcIl4zLjAuMFwiLFxuICAgIFwiQGxpdC9yZWFjdGl2ZS1lbGVtZW50XCI6IFwiXjIuMC4wXCIsXG4gICAgXCJsaXQtZWxlbWVudFwiOiBcIl40LjAuMFwiLFxuICAgIFwibGl0LWh0bWxcIjogXCJeMy4wLjBcIlxuICB9XG59IiwiaGlkZGVuIjp0cnVlfSx7Im5hbWUiOiJjaGlsZC1lbGVtZW50LnRzIiwiY29udGVudCI6ImltcG9ydCB7aHRtbCwgTGl0RWxlbWVudH0gZnJvbSAnbGl0JztcbmltcG9ydCB7Y3VzdG9tRWxlbWVudH0gZnJvbSAnbGl0L2RlY29yYXRvcnMuanMnO1xuXG5AY3VzdG9tRWxlbWVudCgnY2hpbGQtZWxlbWVudCcpXG5leHBvcnQgY2xhc3MgQ2hpbGRFbGVtZW50IGV4dGVuZHMgTGl0RWxlbWVudCB7XG4gIFxuICByZW5kZXIoKSB7XG4gICAgcmV0dXJuIGh0bWxgXG4gICAgICAgIDxwPkNoaWxkIEVsZW1lbnQ8L3A-XG4gICAgICAgIDxzbG90IG5hbWU9XCJ0ZXN0XCI-PC9zbG90PlxuICAgIGA7XG4gIH1cbn1cbiJ9XQ)

**Reply by An-di-K** [reply]
Awesome! 
That is exactly what I was looking for.

Thanks a lot!
